quiz.rb

```diff
@@ -1,20 +1,63 @@
+require 'pry'
 # Question 1
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
+class Dog
+  attr_reader :name
```

Comment:
Tip: you can actually write this in one line by separating the symbols with commas, e.g.
`attr_reader :name, :breed, :age`